src/main/resources/workflowSteps.properties

```diff
@@ -5,7 +5,7 @@
 SUPPORTED=com.day.cq.dam.similaritysearch.internal.workflow.process.AutoTagAssetProcess,com.day.cq.dam.core.impl.process.CreateAssetLanguageCopyProcess,com.day.cq.wcm.workflow.process.CreateVersionProcess,com.day.cq.dam.core.impl.process.TranslateAssetLanguageCopyProcess,com.day.cq.dam.core.impl.process.UpdateAssetLanguageCopyProcess,com.adobe.cq.workflow.replication.impl.ReplicationWorkflowProcess,com.day.cq.dam.core.process.XMPWritebackProcess,com.day.cq.dam.core.process.ScheduledPublishBPProcess,com.day.cq.dam.core.process.ScheduledUnPublishBPProcess,com.day.cq.dam.similaritysearch.internal.workflow.process.InitiateTrainingProcess,com.day.cq.dam.similaritysearch.internal.workflow.process.AddExamplesProcess,com.day.cq.dam.similaritysearch.internal.workflow.process.DoTrainingProcess
```

Comment:
@AbulQasim, I believe, we also need to add them in `SUPPORTED`, correct me if I am wrong @ireasor !

Reply:
Added to SUPPORTED